tier4/driving_log_replayer
# fix(localization): update NDT availability monitoring topic

## Types of PR


## Description

I'd like to update the monitoring topic of NDT availability in localization scenario (merged in https://github.com/tier4/driving_log_replayer/pull/178) so that it can detect runtime errors in some nodes such as `ndt_scan_matcher` or `pointcloud_preprocessor`.

Currently it monitors `/localization/pose_estimator/pose`, but it turned out that it is not suitable for the above application. This is because the topic may not output if the score of NVTL or TP is low, and it is difficult to isolate the cause to the above failure by just monitoring the output.

Thus, I would like to change the monitoring topic to 
- `/localization/pose_estimator/exe_time_ms`

## How to review this PR

1. Run localization scenario to check if all three evaluation passes
2. Run localization scenario with invalid pose initialization (e.g. by setting `1` for [pose initialization particles](https://github.com/autowarefoundation/autoware_launch/blob/47070cb044eaa68949dfeb3a332b0e0c4440610d/autoware_launch/config/localization/ndt_scan_matcher.param.yaml#L38)), and check if the `NDT Availability` evaluation passes. Note that the other two would fail in this case.

## Others
